File: app/src/main/java/com/example/madassignment1/MainActivity.java
package com.example.madassignment1;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main); // Main layout with a "Start" button

        // Find the "Start" button in the main layout
        Button startProfileButton = findViewById(R.id.startProfile);

        // Set up the listener to handle click events
        startProfileButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                // Start the PlayerProfile activity when the button is clicked
                Intent intent = new Intent(MainActivity.this, PlayerProfile1.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/example/madassignment1/MainActivity.kt
package com.example.madassignment1

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_gamemode) // Loads mainpage.xml

        // Find the button by its ID
        val btnPlayAI: Button = findViewById(R.id.btnPlayAI)

        // Set an OnClickListener for the button
        btnPlayAI.setOnClickListener {
            // Create an Intent to start PlayerProfileActivity
            val intent = Intent(this, PlayerProfileActivity::class.java)
            startActivity(intent)
        }
    }
}
